packages/tldraw/src/lib/shapes/shared/useAsset.ts

```diff
@@ -60,13 +62,19 @@ export function useAsset(shapeId: TLShapeId, assetId: TLAssetId | null, width: n
 		}
 
 		// If we already resolved the URL, debounce fetching potentially multiple image variations.
-		const timer = editor.timers.setTimeout(resolve, didAlreadyResolve.current ? 500 : 0)
-
-		return () => {
-			clearTimeout(timer)
-			isCancelled = true
+		if (didAlreadyResolve.current) {
+			const timer = editor.timers.setTimeout(resolve, 500)
```

Comment:
this was originally to get this resolving _before_ we did an SVG export, but it predates the `useDelaySvgExport` stuff. I've reverted and updated this to use that instead. 

fwiw, i think this would only have caused a double resolve in react strict mode in development

Reply:
sadly it wasn't the double resolve in React that was the issue. the problem was that upon page load the Camera started at an initial zoom which propagated to all the shapes. and _then_ once the editor finally loaded it would set the actual initial zoom. this caused two image requests to be happily sent off :-/

Reply:
(unresolving this briefly just so you see it) i'll also add a comment about this in the code

Reply:
i think david's deep link work resolved that - we now initialize the camera at the correct place until waiting for the document to render before moving the camera

Reply:
ah neat. lemme take a look separately from this PR then